test/backend_binary_elementwise.in.cpp

```diff
@@ -148,6 +148,28 @@ NGRAPH_TEST(${BACKEND_NAME}, divide)
     EXPECT_EQ((vector<float>{2, 2, 2, 2}), read_vector<float>(result));
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, divide_int32)
+{
+    Shape shape{2, 2};
+
+    auto A = make_shared<op::Parameter>(element::i32, shape);
+    auto B = make_shared<op::Parameter>(element::i32, shape);
+    auto f = make_shared<Function>(make_shared<op::Divide>(A, B), ParameterVector{A, B});
+
+    auto backend = runtime::Backend::create("${BACKEND_NAME}");
+
+    // Create some tensors for input/output
+    auto a = backend->create_tensor(element::i32, shape);
+    copy_data(a, vector<int32_t>{2, 4, 8, 16});
```

Comment:
These number are easily represented by all datatypes. To exercise int32 you should use numbers that explore a larger space of numbers. Numbers like 0x40000004 are valid int32 numbers but neither bfloat16 nor float can represent it accurately. To test that int32 divide actually works correctly you must test numbers that can't be represented by smaller datatypes.